### DISCUSSION TITLE
Sharing button - Share items or files

### DISCUSSION THREAD

**Discussion by hugopvl** [directus/directus#8084] [Feature Requests]
Hey guys,

Someone asked me if it was possible to share items by email and the attachments that are part of the item (files), but separately, sometimes sharing just the information and sometimes the information plus the attachments.

Attachments are big and won't go through email, so I thought about an integration with wetransfer (I saw now it doesn't support its API anymore) or some service of the kind. Anyone with something similar? Or an extension? 

Well here is the idea :)

**Comment by benhaynes** [comment]
Could tie into this: https://github.com/directus/directus/discussions/2977

Instead of just sharable forms, could be a link to readonly datasets or items. Public makes sense, or with tokens for private info? 🤔

**Reply by hugopvl** [reply]
After I published my idea, I saw your article :) and I think it fits perfectly, send a link to the item or items with read-only access, but with the possibility of downloading the attachments (files).

You can also define in the fields which are "shareable" or not, or select at the time of sharing.

I also came up with the idea of ​​a timer, which can be shared for how long!?

Public makes sence, and tokens for private info makes sense to :) Public forms or public collections, but within the private collections it was handy to have an option to share items, and within the items choose the information to share :) with token :)

**Comment by hugopvl** [comment]
but a question about this feature, [#2977](https://github.com/directus/directus/discussions/2977) will it always be to advance something of the genre or is it still under analysis? :)

Thanks

**Reply by rijkvanzanten** [reply]
I'm not sure what you mean.. What do you mean by "advance something of the genre"?

**Reply by hugopvl** [reply]
I'm talking about Sharable/Public Forms  [#2977](https://github.com/directus/directus/discussions/2977) if you are thinking to go ahead with the idea or if you are just analyzing :)

**Reply by rijkvanzanten** [reply]
Ahh it's definitely something we're interested in adding, but there's quite a lot of R&D that needs to happen before we can go ahead with anything concrete 🙂

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
